web/packages/teleterm/package.json

```diff
@@ -44,18 +43,14 @@
     "@types/node-forge": "^1.0.4",
     "@types/tar-fs": "^2.0.1",
     "@types/whatwg-url": "^11.0.1",
-    "clean-webpack-plugin": "4.0.0",
     "electron": "29.1.4",
     "electron-notarize": "^1.2.1",
     "electron-vite": "^2.0.0",
-    "eslint-import-resolver-webpack": "0.13.2",
```

Comment:
Duplicated in `build/package.json`, but not used. I removed it from both places.